packages/runner-ct/src/app/SpecContent.tsx

```diff
@@ -1,27 +1,27 @@
 import cs from 'classnames'
 import * as React from 'react'
 import SplitPane from 'react-split-pane'
+import { Message, namedObserver, eventManager as EventManager, Header } from '@packages/runner-shared'
 
-import Header from '../header/header'
 import { Iframes } from '../iframe/iframes'
 import { animationFrameDebounce } from '../lib/debounce'
-import { Message } from '../message/message'
 import { KeyboardHelper } from './KeyboardHelper'
 import { NoSpec } from './NoSpec'
 import { Plugins } from './Plugins'
 import { ReporterContainer } from './ReporterContainer'
 import { PLUGIN_BAR_HEIGHT } from './RunnerCt'
 import State from '../lib/state'
-import EventManager from '../lib/event-manager'
 import { hideIfScreenshotting, hideReporterIfNecessary } from '../lib/hideGuard'
 
 import styles from './RunnerCt.module.scss'
-import { namedObserver } from '../lib/mobx'
 
 interface SpecContentProps {
   state: State
   eventManager: typeof EventManager
-  config: Cypress.RuntimeConfigOptions
+  config: {
+    configFile: string
+    [key: string]: unknown
+  }
```

Comment:
This looks like a step backwards with types, is it possible to get Cypress. types here?

Reply:
oh, i assume this is related to your other comment re: runtimeconfigoptions